code/jazmin/python/03_02Grading.py

```diff
@@ -0,0 +1,49 @@
+score = input('please enter your score: ')
+try:
```

Comment:
That does not seem to be running. Make sure to raise a ValueError. Here's an example:

`selection = 1
try:
  choices = [1,2,3,4,5]
  if selection in choices:
    print('Yep, we got it!')
  else:
    raise ValueError
except ValueError:
  print("Not a valid choice!")`

Reply:
@jazzzicodes let me know when this is fixed so that I can approve the PR!

Reply:
oops! I just fixed it, pushing it through right now as 03Grading file

Reply:
@AlexDeMichieli  03Grading is updated with ValueError. Removed 03_02 (version 2 grading lab)
Merged 03_02 code into 03.

Reply:
It seems to be a math issue, but I think we figured that out in a breakout room together? maybe push the latest version,